gulpfile.js

```diff
@@ -48,11 +49,19 @@ function foundation_scss() {
         .pipe(gulp.dest('ynr/assets/scss/'))
 }
 
+function sorttable() {
+    var paths = [
+        `${sorttable}/sorttable.js`,
+    ];
+    return gulp.src(paths)
+        .pipe(gulp.dest('ynr/assets/js/'))
+}
```

Comment:
@symroe What am I missing here? The `sorttable` package doesn't have a `dist` folder which I *think* gulp needs. 😕